README.md

```diff
@@ -4,21 +4,22 @@ Prometheus exporter for Areca RAID cards. Exporter depends on Areca CLI being pr
 
 ## Features
 
-- Provides metrics for the Areca RAID card to be scraped by Prometheus.
+- Provides metrics for the Areca RAID cards to be scraped by Prometheus.
 - Supports the following metrics:
   - `areca_up`: '0' if a scrape of the Areca CLI was successful, '1' otherwise.
-  - `areca_sys_info`: Constant metric with a value of 1 labeled with information about the Areca controller.
+  - `areca_sys_info`: Constant metric with a value of 1 labeled with information about Areca controllers.
   - `areca_raid_set_state`: Areca RAID set state, where 0 represents normal and 1 represents degraded.
-  - `areca_disk_info`: Constant metric with value 1 labeled with info about all physical disks attached to the Areca controller.
+  - `areca_disk_info`: Constant metric with value 1 labeled with info about all physical disks attached to Areca controllers.
   - `areca_disk_state`: Areca controller metric for disk state, 0 for normal, 1 for error
-  - `areca_disk_media_errors`: Metric for media errors of all physical disks attached to the Areca controller.
+  - `areca_disk_media_errors`: Metric for media errors of all physical disks attached to Areca controllers.
```

Comment:
Nice! Maybe we could emphasize that the exporter supports systems with multiple controllers as well?

Reply:
done